src/Identity/Specification.Tests/src/UserManagerSpecificationTests.cs

```diff
@@ -1646,7 +1646,7 @@ public async Task EmailFactorFailsAfterSecurityStampChangeTest()
             var email = await manager.GetUserNameAsync(user) + "@diddly.bop";
             IdentityResultAssert.IsSuccess(await manager.SetEmailAsync(user, email));
             var token = await manager.GenerateEmailConfirmationTokenAsync(user);
-            await manager.ConfirmEmailAsync(user, token);
+            IdentityResultAssert.IsSuccess(await manager.ConfirmEmailAsync(user, token));
```

Comment:
?

Reply:
The first commit had some random flaky test failure. I had a quick look at why it failed and couldn't fathom why (didn't seem related to my change either), but while I was looking I noticed that those two tests weren't asserting that this step was actually succeeding.

Reply:
And then of course the second build failed for some random other reason 😅